Source/OxyPlot/Series/HeatMapSeries.cs

```diff
@@ -122,6 +122,21 @@ public HeatMapSeries()
         public bool Interpolate { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to use discrete rectangles when rendering (ignored if Interpolate is set to true). The default value is <c>false</c>.
+        /// </summary>
+        public bool RenderDiscreteRectangles { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to treat the x dimension as logarithmic. The default value is <c>false</c>.
+        /// </summary>
+        public bool LogX { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to treat the y dimension as logarithmic. The default value is <c>false</c>.
+        /// </summary>
+        public bool LogY { get; set; }
```

Comment:
It was suggested in a previous PullRequest that it may be possible to detect these automatically from the axes configuration. I am dubious of this, in part because of the existence of #164 (which these changes make no effort to address). I can also not think of a nice way of determining if the axes are logarithmic or not.